test/replicate.test.js

```diff
@@ -4,7 +4,6 @@ const Keys = require('ssb-keys')
 const { Server, replicate } = require('./helpers')
 
 test('replicate group members', async t => {
-
```

Comment:
This whole file might? Be able to be deleted. Need to check

Reply:
I'm not really sure what's being tested (installReplicate and persistance when restarting server?) so I'll leave it to you to decide.  I do note that I somehow  missed adding live:true to this replicate when I ctrl-shift-F-ed and yet it still seems to pass fine.

Reply:
yeah got it. Figured out what this test was for and will leave clearer notes for future explorerz